javascript/webpacker_react-npm-module/package.json

```diff
@@ -13,5 +13,8 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.0"
+  },
+  "dependencies": {
+    "lodash": "^4.0.0"
```

Comment:
I would prefer to not depend on lodash, especially if we only use 1 or 2 function